test/0.8.9/staking-router-keys-reporting.test.js

```diff
@@ -482,31 +514,34 @@ contract('StakingRouter', ([deployer, lido, admin]) => {
         assert.equal(callInfo2.onExitedAndStuckValidatorsCountsUpdated.callCount, 0)
       })
 
-      it(`calling onValidatorsCountsByNodeOperatorReportingFinished doesn't call ` +
-         `anything on any module`, async () =>
-      {
-        await router.onValidatorsCountsByNodeOperatorReportingFinished({ from: admin })
+      it(
+        `calling onValidatorsCountsByNodeOperatorReportingFinished doesn't call ` + `anything on any module`,
```

Comment:
don't need `+` here and in similar places anymore

Reply:
also, the fact that a long test name idents it more than other tests at the same level but with shorter names significantly complicates visual navigation through the file, imo

Reply:
Tried to fix it.